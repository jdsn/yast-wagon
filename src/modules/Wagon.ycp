/**
 * File:	modules/Wagon.ycp
 * Package:	WAGON
 * Summary:	Functions and handlers for WAGON
 * Authors:	Lukas Ocilka <locilka@suse.cz>
 * Internal
 *
 * $Id$
 *
 * Module for handling WAGON.
 *
 */

{
    module "Wagon";

    textdomain "wagon";

    import "Installation";
    import "PackageCallbacks";

    /**
     * Initializes the package manager
     */
    global boolean InitPkg () {
	if (Pkg::TargetInitialize (Installation::destdir) != true) {
	    y2error ("Cannot initialize target");
	    return false;
	}

	if (Pkg::SourceStartManager (true) != true) {
	    y2error ("Cannot initialize sources");
	    return false;
	}

	return true;
    }

/* EOF */
}
