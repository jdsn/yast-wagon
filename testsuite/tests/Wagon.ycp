{

// testedfiles: Wagon.ycp

include "testsuite.ycp";
TESTSUITE_INIT([], nil);

import "Wagon";

DUMP("Wagon::Modified");
TEST(``(Wagon::Modified()), [], nil);

}
