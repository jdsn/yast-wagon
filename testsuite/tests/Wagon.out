Dump	Wagon::Modified
Return	false
